content/docs/user-guide/project-structure/configuration.md

```diff
@@ -378,6 +379,22 @@ Composition].
 
 <details>
 
+## studio
+
+- `studio.token` - Studio access token to use. When this is set, DVC uses this
+  to notify Studio of new experiments, and provide live updates of metrics and
+  plots to Studio. Alternatively, this can also be specified through
```

Comment:
I added `provide live updates` since we have plans to. Let me know if you want to skip it for now.

Reply:
Let's skip it 🙏